spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/ResNet50Perf.scala

```diff
@@ -49,15 +49,21 @@ object ResNet50Perf {
   }
 
   def main(argv: Array[String]): Unit = {
+    System.setProperty("bigdl.disable.mklBlockTime", "true");
     System.setProperty("bigdl.mkldnn.fusion.convbn", "true")
     System.setProperty("bigdl.mkldnn.fusion.bnrelu", "true")
     System.setProperty("bigdl.mkldnn.fusion.convrelu", "true")
     System.setProperty("bigdl.mkldnn.fusion.convsum", "true")
 
-    val coreNumber: Int = Runtime.getRuntime.availableProcessors() / 2
-    System.setProperty("bigdl.mklNumThreads", s"$coreNumber")
-    Engine.setCoreNumber(1)
-    MklDnn.setNumThreads(coreNumber)
+//    val coreNumber: Int = System.getProperty("bigdl.mklNumThreads",
+//      s"${Math.ceil(Runtime.getRuntime.availableProcessors() / 2).toInt}").toInt
+    System.setProperty("bigdl.localMode", "true")
```

Comment:
Can we use one line e.g. System.setProperty("mkldnn", "true") to replace so many lines?